Arslan.cpp

```diff
@@ -0,0 +1,13 @@
+#include<iostream>
+using namespace std;
```

Comment:
Bad practice to write `using namespace any_namespace;`. You should explicitly use the namespace methods like `std::endl`. You should delete this line.